### ISSUE TITLE
[Feature Improvement] Radio Refactor Pt 2 - Quality of Life Improvements

### ISSUE THREAD

**Issue by dgibso29** [Bubbus/F3_CA_BUB#30]
- [ ] 🔴  try to preserve radio spatialization, PTT slot, volume

Bub - deleted all the rest as no longer relevant (markers radio list, hotline zeus)

**Comment by Bubbus** [comment]
These are stretch goals on the back of #133.  IMO "Hotline Zeus" should be axed as it's barely used these days and players have become more accustomed to pinging for problems.  Maybe replace with a standard disclaimer of "ping if shits fucked or shits unfun".

**Comment by dgibso29** [comment]
Agree, we had a disclaimer in our collabs for a while that feels like a decent fit.

**Comment by Bubbus** [comment]
Moving to backlog - need to get radios stable before adding on QoL
